### PR TITLE
✅ [amp-google-read-aloud-player] Adds validator and examples for `amp-google-read-aloud-player` extension

### PR DESCRIPTION
**Pull request by mhalabi-google**
See [I2I #31013](https://github.com/ampproject/amphtml/issues/31013).

cc: @alanorozco 
cc: @MichaelRybak

### CODE REVIEW

File: extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  tag_name: "AMP-GOOGLE-READ-ALOUD-PLAYER"
+  requires_extension: "amp-google-read-aloud-player"
+  attrs: {
+    name: "data-api-key"
+    mandatory: true
+  }
+  attrs: {
+    name: "data-tracking-ids"
+    mandatory: true;
+    value_regex_casei: "^UA-\\d+-\\d+(\\s*,\\s*UA-\\d+-\\d+)*";
```
Comment (MichaelRybak) [inline extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii:22]: Does this regexp need a $ at the end?

**Review by MichaelRybak — COMMENTED** [review]
Looks good! Optionally add a few more tags into html that test the attributes below:

* data-locale
* data-ad-tag-url and it's value_url params
* an http value for for data-url
* a few more tests for the data-tracking-ids regexp:
  UA-1-123123
  ua-1-1
  123
  ua-1
  UA-1-1.2
  (empty string)
  UA-1-1 XYZ

**Comment by MichaelRybak** [comment]
@alanorozco Alan would you mind reviewing the example html, as well as giving a quick look to all the files too just in case? It's my first time reviewing a new extension.

File: extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  }
+  attrs: {
+    name: "data-ad-tag-url"
+    value_url {
+      protocol: 'https';
+      protocol: 'http';
+      allow_relative: false;
+    }
+  }
+  spec_url: "https://amp.dev/documentation/components/amp-google-read-aloud-player"
```
Comment (MichaelRybak) [inline extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii:68]: Would it be a good idea to refer to documentation for Tracking ID from the component documentation? E.g. to this: https://support.google.com/analytics/answer/7372977?hl=en

File: extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii (hunk 1 above)
Comment (mhalabi-google) [reply extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii:22]: Yes. Added.

**Comment by mhalabi-google** [comment]
> Looks good! Optionally add a few more tags into html that test the attributes below:
> 
> * data-locale
> * data-ad-tag-url and it's value_url params
> * an http value for for data-url
> * a few more tests for the data-tracking-ids regexp:
>   UA-1-123123
>   ua-1-1
>   123
>   ua-1
>   UA-1-1.2
>   (empty string)
>   UA-1-1 XYZ

Done. Thank you for the suggestions!

**Review by mhalabi-google — COMMENTED** [review]
Thank you for the quick review!

File: extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    value_url {
+      protocol: 'https';
+      allow_relative: false;
+    }
+  }
+  attrs: {
+    name: "data-outro"
+    value_url {
+      protocol: 'https';
+      protocol: 'http';
```
[the remark this answers is not in the document]
Comment (mhalabi-google) [reply extensions/amp-google-read-aloud-player/validator-amp-google-read-aloud-player.protoascii]: Done.

**Review by MichaelRybak — APPROVED** [review]

**Comment by mhalabi-google** [comment]
Thank you @MichaelRybak and @alanorozco for the review!

**Comment by alanorozco** [comment]
@mhalabi-google Are you satisfied with the rules and test results? If so I'll merge

**Comment by luissegovia** [comment]
Alguien que me ayude a como implementar esto? estoy super perdido...

de donde genero el API KEY?

**Pull request merged**